artman/cli/main.py

```diff
@@ -179,6 +179,11 @@ def parse_args(*args):
         type=str,
         help='[Required] Name of the artifact for artman to generate. Must '
         'match an artifact in the artman config yaml.')
+    parser_generate.add_argument(
+        '--aspect',
```

Comment:
Are you sure that aspect is a proper name here? It feels too generic. I knew what this review would be about, but even in this case i was baffled by "aspect" argument. Maybe something more specific like "artifact_type" or something like that?

Reply:
I tried out the `artifact_type` name initially, but then it felt like it was too easy to confuse with `artifact_name`. Any other ideas? Maybe `artifact_aspect`?

Reply:
Definitelly not `artifact_aspect` it is even worse. How is coexistence of `name` and `type` is confusing? It is a very-very common pair of properties.

Reply:
Definitely not `artifact_aspect` it is even worse. How is coexistence of `name` and `type` confusing? It is a very-very common pair of properties.

Reply:
It's confusing because `artifact_name` and `artifact_type` where wouldn't be related in the same way as the name of a variable and the type of a variable are related. The existing `artifact_name` is actually a type, and would actually be better named if it was `artifact_type`. The term `aspect` here is meant to invoke the concept of aspect-oriented programming, indicating that it is a cross-cutting concern.